flink-ml-servable-core/src/test/java/org/apache/flink/ml/servable/builder/ExampleServables.java

```diff
@@ -0,0 +1,99 @@
+ */
+
+package org.apache.flink.ml.servable.builder;
+
+import org.apache.flink.api.common.typeutils.base.IntSerializer;
+import org.apache.flink.core.fs.FSDataInputStream;
+import org.apache.flink.core.fs.Path;
+import org.apache.flink.core.memory.DataInputViewStreamWrapper;
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.servable.api.DataFrame;
+import org.apache.flink.ml.servable.api.Row;
+import org.apache.flink.ml.servable.api.TransformerServable;
+import org.apache.flink.ml.servable.types.DataTypes;
+import org.apache.flink.ml.util.FileUtils;
+import org.apache.flink.ml.util.ParamUtils;
+import org.apache.flink.ml.util.ServableReadWriteUtils;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/** Defines Servable subclasses to be used in unit tests. */
+public class ExampleServables {
+
+    /**
+     * A {@link TransformerServable} subclass that increments every value in the input dataframe by
+     * `delta` and outputs the resulting values.
+     */
+    public static class SumModelServable implements TransformerServable<SumModelServable> {
+
+        private static final String COL_NAME = "input";
+
+        private final Map<Param<?>, Object> paramMap = new HashMap<>();
+
+        private int delta;
+
+        public SumModelServable() {
+            ParamUtils.initializeMapWithDefaultValues(paramMap, this);
+        }
+
+        @Override
+        public DataFrame transform(DataFrame input) {
+            List<Row> outputRows = new ArrayList<>();
+            for (Row row : input.collect()) {
+                assert row.size() == 1;
+                int originValue = (Integer) row.get(0);
+                outputRows.add(new Row(Collections.singletonList(originValue + delta)));
+            }
+            return new DataFrame(
+                    Collections.singletonList(COL_NAME),
+                    Collections.singletonList(DataTypes.INT),
+                    outputRows);
+        }
+
+        @Override
+        public Map<Param<?>, Object> getParamMap() {
+            return paramMap;
+        }
+
+        public static SumModelServable load(String path) throws IOException {
+            SumModelServable servable =
+                    ServableReadWriteUtils.loadServableParam(path, SumModelServable.class);
+
+            Path modelDataPath = FileUtils.getDataPath(path);
+            try (FSDataInputStream fsDataInputStream =
+                    FileUtils.getModelDataInputStream(modelDataPath)) {
+                DataInputViewStreamWrapper dataInputViewStreamWrapper =
+                        new DataInputViewStreamWrapper(fsDataInputStream);
+                int delta = IntSerializer.INSTANCE.deserialize(dataInputViewStreamWrapper);
+                servable.setDelta(delta);
+            }
+            return servable;
+        }
+
+        public SumModelServable setDelta(int delta) {
```

Comment:
Should we replace this method with `SumModelServable#setModelData(...)` so that users can set the model data using the public API designed in FLIP-289?

Reply:
I'm not sure if I got your point. I add `setDelta` method just to construct multiple different `Transformer`s and chain them into a pipeline, then perform the `saveLoadAndTransform` tests. It is simpler with a setter rather than a `setModelData`.